### ISSUE TITLE
Bug with block contry

### ISSUE THREAD

**Issue by thelittlefireman** [bunkerity/bunkerized-nginx#27]
hi,

With the following configuration i've got this error :

```
[*] Starting bunkerized-nginx ...
sed: unmatched '/'
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator standalone, Installer None
Unable to register an account with ACME server
[*] Updating clamav (in background) ...
time="2020-11-11T20:21:03+01:00" level=info msg="api load configuration: configuration loaded successfully (base:https://tmsov6x2n9.execute-api.eu-west-1.amazonaws.com/v1/)"
time="2020-11-11T20:21:12+01:00" level=info msg="api load configuration: configuration loaded successfully (base:https://tmsov6x2n9.execute-api.eu-west-1.amazonaws.com/v1/)"
time="2020-11-11T20:21:12+01:00" level=info msg="dependency issue crowdsecurity/nginx : missing parsers crowdsecurity/nginx-logs, tainted."
time="2020-11-11T20:21:13+01:00" level=info msg="api signin: signed in successfuly"
time="2020-11-11T20:21:15+01:00" level=warning msg="api pull returned 100 entries"
time="2020-11-11T20:21:26+01:00" level=info msg="Wrote 100 bans from api to database."
[*] Running nginx ...
nginx: [emerg] unexpected "}" in /etc/nginx/geoip.conf:10
==> /var/log/access.log <==

==> /var/log/error.log <==

==> /var/log/fail2ban.log <==
2020-11-11 20:21:26,822 fail2ban.filter         [12799]: INFO      maxRetry: 15
2020-11-11 20:21:26,822 fail2ban.filter         [12799]: INFO      findtime: 60
2020-11-11 20:21:26,823 fail2ban.actions        [12799]: INFO      banTime: 3600
2020-11-11 20:21:26,823 fail2ban.filter         [12799]: INFO      encoding: UTF-8
2020-11-11 20:21:26,823 fail2ban.filter         [12799]: INFO    Added logfile: '/var/log/access.log' (pos = 0, hash = da39a3ee5e6b4b0d3255bfef95601890afd80709)
2020-11-11 20:21:26,825 fail2ban.jail           [12799]: INFO    Jail 'nginx-filter' started
2020-11-11 20:21:26,881 fail2ban.utils          [12799]: #39-Lev. 7fa7c0bfac30 -- exec: echo "" > /etc/nginx/fail2ban-ip.conf && /usr/sbin/nginx -s reload
2020-11-11 20:21:26,881 fail2ban.utils          [12799]: ERROR   7fa7c0bfac30 -- stderr: 'nginx: [emerg] unexpected "}" in /etc/nginx/geoip.conf:10'
2020-11-11 20:21:26,881 fail2ban.utils          [12799]: ERROR   7fa7c0bfac30 -- returned 1
2020-11-11 20:21:26,882 fail2ban.actions        [12799]: ERROR   Failed to start jail 'nginx-filter' action 'nginx-action': Error starting action Jail('nginx-filter')/nginx-action
^C

```

```
version: '3.5'

services: 
  nginx_test:
    image: bunkerity/bunkerized-nginx
    container_name: nginx_test
    restart: always
    network_mode: "host"
    ports:
      - 10080
      - 10443
    volumes:
      - ./nginx_test/www:/www
      - ./nginx_test/letsencrypt:/etc/letsencrypt
      - ./nginx_test/server-confs:/server-confs         # custom nginx confs at server context to make Nextcloud working
      - ./nginx_test/modsec-crs-confs:/modsec-crs-confs # custom Core Rule Set confs to add Nextcloud exclusions
      - ./nginx_test/modsec-confs:/modsec-confs         # disable some false positive
      # Set timezone
      - "/etc/TZ:/etc/timezone:ro"
      - "/etc/localtime:/etc/localtime:ro"
    environment:
      - SERVER_NAME="*REDACTED"          # replace with your domain
      - SERVE_FILES=yes
      - HTTP_PORT=10080
      - HTTPS_PORT=10443
      - PROXY_REAL_IP=yes
      - PROXY_REAL_IP_HEADER=X-Forwarded-For
      - PROXY_REAL_IP_RECURSIVE
      - USE_CROWDSEC=yes
      - AUTO_LETS_ENCRYPT=yes
      - REDIRECT_HTTP_TO_HTTPS=yes
      - DISABLE_DEFAULT_SERVER=yes
      - MAX_CLIENT_SIZE=10G
      - LIMIT_REQ_RATE=40r/s
      - LIMIT_REQ_BURST=60
      - LIMIT_REQ_CACHE=50m
      - BLOCK_COUNTRY="cn ru"
      - ALLOWED_METHODS=GET|POST|HEAD|PROPFIND|DELETE|PUT|MKCOL|MOVE|COPY|PROPPATCH|REPORT
      - X_FRAME_OPTIONS=SAMEORIGIN
     

```

**Comment by bunkerity** [comment]
Thanks for your report, fix will be integrated in the next version. Please note that you need to remove the double quote (") when setting environment variables inside your docker-compose.yml.

**Comment by thelittlefireman** [comment]
Perfect thanks you :+1:

**Comment by bunkerity** [comment]
Hello @thelittlefireman, this is now fixed in the [1.2.0](https://github.com/bunkerity/bunkerized-nginx/releases/tag/v1.2.0) version. Please note that you can define whitelist or blacklist through the `WHITELIST_COUNTRY` and `BLACKLIST_COUNTRY` environment variables. The `BLOCK_COUNTRY` is not working anymore, use `BLACKLIST_COUNTRY` instead to achieve the same goal. More info [here](https://github.com/bunkerity/bunkerized-nginx#countries).
